airflow/auth/managers/fab/auth/__init__.py

```diff
@@ -15,3 +14,252 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from __future__ import annotations
+
+import datetime
+
+# This product contains a modified portion of 'Flask App Builder' developed by Daniel Vaz Gaspar.
+# (https://github.com/dpgaspar/Flask-AppBuilder).
+# Copyright 2013, Daniel Vaz Gaspar
```

Comment:
is that ok in open source code ?

Reply:
I am just moving code within Airflow codebase, so I guess there is no change here

Reply:
oh ok didn't realize it was already there, I thought it was new code.